docs/querying_history.rst

```diff
@@ -175,30 +175,49 @@ model history.
     <Poll: Poll object as of 2010-10-25 18:04:13.814128>
 
 
-Save without a historical record
---------------------------------
+Save Without Creating Historical Records
+----------------------------------------
 
-If you want to save a model without a historical record, you can use the following:
+If you want to save model objects without triggering the creation of any historical
+records, you can set a ``skip_history_when_saving`` attribute to ``True``
+on each object before saving - for example like this:
```

Comment:
Personally, I think the previous version is better. It's fewer words and you have a code example to show the how.

Reply:
Good point on already having a code example 😅 I'll push a slightly shorter version 🙂 I do prefer being specific about "triggering the creation", though.